pkg/skaffold/gcp/auth.go

```diff
@@ -64,6 +64,8 @@ func activeUserCredentials() (*google.Credentials, error) {
 		cmd := exec.Command("gcloud", "auth", "print-access-token", "--format=json")
 		body, err := util.RunCmdOut(cmd)
 		if err != nil {
+			logrus.Infof("unable to retrieve gcloud access token: %v", err)
+			logrus.Infof("falling back to application default credentials")
 			credsErr = fmt.Errorf("retrieving gcloud access token: %w", err)
```

Comment:
Should this instead be an Actionable Error? `sErrors.Error`?

Reply:
Probably not, since we fall through to the client library, which should default to the ADC?  The underlying error does show an actionable error from gcloud.  This should really be be a warning, not info.

Reply:
```
Starting build...
Building [skaffold-example]...
INFO[0002] unable to retrieve gcloud access token: running [gcloud auth print-access-token --format=json]
 - stdout: ""
 - stderr: "ERROR: (gcloud.auth.print-access-token) There was a problem refreshing your current auth tokens: ('invalid_grant: Invalid grant: account not found', {'error': 'invalid_grant', 'error_description': 'Invalid grant: account not found'})\nPlease run:\n\n  $ gcloud auth login\n\nto obtain new credentials.\n\nIf you have already logged in with a different account:\n\n    $ gcloud config set account ACCOUNT\n\nto select an already authenticated account to use.\n"
 - cause: exit status 1 
INFO[0002] falling back to application default credentials 
```

It's not pretty though.

Reply:
`gcloud` isn't outputting the error as JSON; filed b/191668532 to track.